### PR TITLE
Colors for different battery levels

### PR DESCRIPTION
**Pull request by haoxiangliew**
Current levels are set to 10% (Red) 50% (Yellow) and 100% (Green)

### CODE REVIEW

File: src/workspacer.Bar/Widgets/BatteryWidget.cs
Comment (josteink) [inline src/workspacer.Bar/Widgets/BatteryWidget.cs:19]: This value isn't used for anything and should be removed. 

Apart from that, this all looks good to me 👍

**Review by josteink — CHANGES_REQUESTED** [review]

File: src/workspacer.Bar/Widgets/BatteryWidget.cs
Comment (josteink) [inline src/workspacer.Bar/Widgets/BatteryWidget.cs:39]: Same comment as last time: we don't need high charge threshold, and this else-is block should be merged with the following else-block.

**Review by josteink — CHANGES_REQUESTED** [review]

File: src/workspacer.Bar/Widgets/BatteryWidget.cs
Comment (haoxiangliew) [reply src/workspacer.Bar/Widgets/BatteryWidget.cs]: Changed it to only have a Low Battery Color

**Comment by josteink** [comment]
I think maybe you removed too much? This is the entire PR now: :smile: 

image

**Comment by haoxiangliew** [comment]
Just changed the file, and removed the redundant else function. Already in stable is the low battery color (red), my commit only adds two levels of color to the battery widget for medium battery (less than 50%) and high battery (above 50%)

**Comment by haoxiangliew** [comment]
Also the new layouts are in another PR now :D

**Comment by josteink** [comment]
Looks much better!

Before we merge, could you please remove the "HighChargeThreshold" variable, since it is no longer in use :)

**Comment by haoxiangliew** [comment]
Done!

**Comment by josteink** [comment]
LGTM!

Thanks for the patch!

**Review by josteink — APPROVED** [review]

**Comment by haoxiangliew** [comment]
No problem. thanks for reviewing my PR!

**Comment by rickbutton** [comment]
thanks for the PR @haoxiangliew and thanks for reviewing @josteink

**Comment by josteink** [comment]
Np, Rick. 

As for the layouts PR, I'll leave that to someone else. Don't think that's my area of expertise 😉

**Pull request merged**
